src/js/conntest.js

```diff
@@ -40,32 +40,62 @@ function hostConnectivityTest() {
 }
 
 function runConnectivityTest(iceCandidateFilter, config) {
+  var candidates = [];
   var call = new Call(config);
   call.setIceCandidateFilter(iceCandidateFilter);
+
+  // Collect all candidate for validation.
+  call.pc1.onicecandidate = function(event) {
+    if (event.candidate) {
+      candidates.push(Call.parseCandidate(event.candidate.candidate));
```

Comment:
I want to gather all candidates in case we want to report different messages for different candidate types, otherwise the `candidates` array should probably be named reflexiveCandidates instead.